cmd/delete.go

```diff
@@ -28,7 +28,7 @@ No local files are deleted.
 {{rootCmdUse}} delete
 
 # Undeploy the function 'myfunc' in namespace 'apps'
-{{rootCmdUse}} delete -n apps myfunc
+{{rootCmdUse}} delete myfunc --namespace apps
```

Comment:
does this need release notes? Mirgation? or the like?

Reply:
These two commands are equivalent.  I was just making the documentation use the long-form name for (hopefully) better clarity.